example/storybook/src/components/Forms/Button/index.stories.mdx

```diff
@@ -477,6 +688,145 @@ function App () {
   />
 </AppProvider>
 
+#### Button with Full Width
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+        <Wrapper>
```

Comment:
remove the Wrapper. Don't use Wrapper when you are using transformedCode function